scripts/release/generate_history_notes.py

```diff
@@ -169,7 +170,7 @@ def get_prs_for_commit(commit: str):
 
 def process_pr(pr):
     lines = [pr['title']]
-    body = pr['body']
+    body = pr['body'] or ''
```

Comment:
When a PR doesn't have a body, GitHub API returns `None` instead of `''`. This breaks the script.